src/Faker/Provider/et_EE/Person.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+namespace Faker\Provider\et_EE;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $titleMale = array('Härra', 'Hr.');
+
+    protected static $titleFemale = array('Proua', 'Pr.', 'neiu', 'preili');
+
+    /**
+     * @link http://www.eki.ee/books/ekk09/index.php?p=6&p1=5&id=548
```

Comment:
This page mentions a copyright

Reply:
> No, read the contributing guide again: you must provide a reference for a MIT-compatible list of names

the contribution guide only refers that the software must be MIT-compliant. I do understand that this also extends to information contained within, but this sounded a bit like the name lists must have a link, which they must not (and MANY existing usages do not). There is no such obligation, nor should there be.

> A name cannot be copyrighted, but a list of names can, believe me.

I've never been much of a believer. If you can actually show me Estonian law that confirms that a list of names is ownable by means of copyright, I'll believe. For now, I examined the laws myself and saw no such possibility.

All that aside, I can't even find the list on that page. To me it seems, the book was only a partial reference (which means there is NO WAY this list is under any kind of copyright). If you're still worried, I could always add a few names from the local phonebook, off the top of my head or from http://nimi.ee/poiste-nimed/a/ , a  page that offers names, by letter, without any copyright.

Reply:
I appreciate the work you've done on the other formatters, but I can't merge your PR because of this. If you want your PR to be merged to Faker, please find a datasource compatible with MIT, or remove the Person provider.